core/src/main/scala/com/salesforce/op/OpWorkflowCore.scala

```diff
@@ -306,7 +307,9 @@ private[op] trait OpWorkflowCore {
   def computeDataUpTo(feature: OPFeature, path: String)
     (implicit spark: SparkSession): Unit = {
     val df = computeDataUpTo(feature)
-    df.saveAvro(path)
+    JobGroupUtil.withJobGroup(OpStep.SavingScores) {
```

Comment:
These are not necessarily scores, but ok.

Reply:
renamed